src/System.Numerics.Vectors.WindowsRuntime/src/System/Numerics/VectorExtensions.cs

```diff
@@ -1,54 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can't WinRT be used from outside of app containers / uap?

Reply:
Yes. Many of the WinRT APIs including most of the WinRT networking APIs (Windows.Networking.*, Windows.Web.*) can be used from "Desktop" applications. And supposedly they can be used now with .NET Core 3.0 because it has "Desktop" support.

Reply:
Yeah.  Given that, I don't think we should be removing this assembly.

Reply:
This assembly has 4 trivial extension helper methods in it. I do not think we would want to ship a whole new file with 4 methods in .NET 5. If these methods are really needed, we should find a new home for them in one of the existing .dlls in .NET 5.

Reply:
If we're not already shipping it, then it's fine to remove it.